resources/wf_builder.rb

```diff
@@ -0,0 +1,290 @@
+#
+# Author:: Nathan Cerny <ncerny@chef.io>
+#
+# Cookbook Name:: chef_stack
+# Resource:: wf_runner
+#
+# Copyright 2017 Chef Software Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource_name 'workflow_builder'
```

Comment:
Looking further down the resource, I wonder if we shouldn't split out this resource as `workflow_builder` for v1 and `workflow_runner` for v2.